CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+* Adds Reset Password feature to the login page. See the [documentation](https://v3.docs.apostrophecms.org/) for more details.
```

Comment:
May as well add "Note that the feature must be enabled and email delivery must be properly configured" here, just so people don't try to skip the documentation.